OracleWebLogic/samples/12213-webtier-apache/Dockerfile

```diff
@@ -1,61 +1,92 @@
 # Example of Apache HTTP Server with WebLogic plugin for load balancing WebLogic on Docker Containers
 #
-# Copyright (c) 2016-2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2016-2020 Oracle and/or its affiliates.
 #
-# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 #
 # REQUIRED FILES TO BUILD THIS IMAGE
 # ----------------------------------
-# fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip
+# For version 12.2.1.3 Required file fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip
+# For Version 12.2.1.4 Required file fmw_12.2.1.4.0_wlsplugins_Disk1_1of1.zip
```

Comment:
For Version 12.2.1.4 Required file -> For version 12.2.1.4, the required file is